packages/runtime/src/interpreter/opcode-list.ts

```diff
@@ -4355,21 +4360,44 @@ export class ITxn extends Op {
 			});
 		}
 
-		let result;
-		// what is "last "
+		let result: StackElem;
+		// what is "last"
 		const groupTx = this.interpreter.innerTxnGroups[this.interpreter.innerTxnGroups.length - 1];
 		const tx = groupTx[groupTx.length - 1];
 
 		switch (this.field) {
 			case "Logs": {
```

Comment:
Can you explain for me about the logs? I have read the document but I did not get it through